amida-tech/indaba-client
# Feature/inba 957 grommet checkbox removal

#### What does this PR do?

Removes the Grommet Checkbox component in preparation for removing Grommet from Indaba completely. Replaces the two instances of Grommet checkboxes with custom checkbox components written in JSX.

#### Related JIRA tickets:

INBA-957

#### How should this be manually tested?

1) Checkout the repo and start up your indaba services and indaba-client.
2) Sign into a project management user
3) Navigate to the user's profile page at /profile, and check that their profile contains a non-functional checkbox under the heading "Is active?"
4) Navigate to any project where a user is assigned to a stage, and that stage has not yet been completed. 
5) Click on the three ellipses next to the user's name in the task assignment.
6) Once there, you should see two radio buttons and a checkbox. Click on the checkbox and check the console to make sure it's modifying the state, and make sure that the checkbox turns black when you hover over it. 


#### Background/Context

#### Screenshots (if appropriate):
![capture](https://user-images.githubusercontent.com/35747608/50905951-03e71500-13f2-11e9-9fd8-d19144ed025d.PNG)
